backend/src/main.rs

```diff
@@ -50,22 +49,22 @@ async fn main() {
 
     let file = File::open(args.config).unwrap_or_else(|err| {
         error!("failed to open config file: {}", err);
-        std::process::exit(1);
+        std::process::exit(2);
     });
 
     let yaml_config: Arc<YamlConfig> = match serde_yaml::from_reader(file) {
         Ok(config) => Arc::new(config),
         Err(err) => {
             error!("failed to parse config file: {}", err);
-            std::process::exit(1);
+            std::process::exit(2);
```

Comment:
What's the plan for using exit codes > 1?

Reply:
It was to determine what was the cause of the crash. I guess we can assign an error code for each reason?

Reply:
From my experience it's hard to maintain and requires a lot of effort - it's much better just sticking to one status code and returning a proper error message.

Reply:
Understood :)